acr-build.yml

```
@@ -2,34 +2,67 @@ version: v1.1.0
 steps:
   - id: ccd
     build: >
-      -t {{.Run.Registry}}/ccd/logstash:ccd-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-latest -t {{.Run.Registry}}/ccd/logstash:latest
+      -t {{.Run.Registry}}/ccd/logstash:ccd-logstash-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-logstash-latest -t {{.Run.Registry}}/ccd/logstash:latest
       --build-arg conf=ccd_logstash.conf.in
       .
     keep: true
 
   - id: probate
```

```suggestion
  - id: ccdprobate
```
see: https://github.com/hmcts/ccd-logstash/blob/b0210df9a3b5b59ad8d5470bf26fe89eade001ec/Jenkinsfile_CNP#L10